frontend/packages/console-app/src/components/quick-starts/data/add-healthchecks-quickstart.ts

```diff
@@ -0,0 +1,74 @@
+export const addHealthChecksQuickStart = {
+  apiVersion: 'console.openshift.io/v1',
+  kind: 'QuickStarts',
+  metadata: {
+    name: 'add-healthchecks',
+  },
+  spec: {
+    version: 4.7,
+    displayName: 'Adding health checks to your sample application',
+    duration: 10,
+    iconURL: '',
+    description: `You just created a sample application. Now, let’s add health checks to it.`,
+    prerequisites: `You completed the "Getting started with a sample" quick start.`,
+    introduction: `### This quick start shows you how to add health checks to your sample application.
+You should have previously created the **sample-app** application and **nodejs-sample** deployment via the **Get started with a sample** quick start. If you haven't, you may be able to follow these tasks with any existing deployment without configured health checks.`,
```

Comment:
suggest replacing 'via' with 'using'

Reply:
+1.

I think that it's odd to to have a line that essentially says "you might be able to do this, or you might not," too.